code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -10325,3 +10325,29 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 			new /obj/effect/alien/weeds(T)
 		if(!locate(/obj/effect/decal/cleanable/purpledrank) in T)
 			new /obj/effect/decal/cleanable/purpledrank(T)
+
+/datum/reagent/punchualite
+	name = "Punchualite"
+	id = PUNCHUALITE
+	description = "Nicknamed mad chemist's alarm clock. Explodes on the hour When being actively metabolized. "
```

Comment:
Capitalization.

Reply:
That's what I get for rewriting sentences so much